flask_smorest/utils.py

```diff
@@ -23,6 +23,11 @@ def deepupdate(original, update):
     return original
 
 
+def remove_none(mapping):
+    """Remove None values in a dict"""
+    return {k: v for k, v in mapping.items() if v is not None}
```

Comment:
You didn't test it.
```
def test_remove_none():
    mapping = {"a": 1, "b": None, "c": 3, "d": None, e: "None", "f": 6}
    
    result = remove_none(mapping)

    assert result == {"a": 1, "c": 3", "f": 6}
```

Reply:
No. Technically, the test coverage remains the same, it is just a refactor.

But sure, since this util function is now available to test, I could add an extra test.